crates/wasmtime/src/instance.rs

```diff
@@ -82,18 +82,29 @@ fn instantiate(
                 imports.modules.push(module.submodule(*idx));
             }
 
-            // Here we lookup our instance handle, ask it for the nth export,
+            // Here we lookup our instance handle, find the right export.
```

Comment:
GitHub needs a wee lil' nit emoji:

```suggestion
            // Here we lookup our instance handle, find the right export,
```